backend/mr-db-sql/src/main/java/io/featurehub/db/model/DbPerson.java

```diff
@@ -28,6 +29,12 @@
   @Version
   private long version;
 
+  public static final String DEFAULT_PASSWORD_ALGORITHM = "PBKDF2WithHmacSHA512";
+
+  @Column(name = "password_alg", length = 60, nullable = false)
+  @DbDefault("PBKDF2WithHmacSHA1")
```

Comment:
Yes, because anyone with the existing database will have the SHA1 password. It will just create the field in the database with this value already set, given it is nullable false. Otherwise i would have to set it nullable true, issue an update for everyone and still have the same issue with forcing the change for new accounts or logins.

Reply:
okay got it